hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystem.java

```diff
@@ -73,15 +73,17 @@ public boolean apply(FileStatus status) {
      * Key mapping to the public/private key algorithm (ex: RSA).
      */
     public static final String KEY_ALGORITHM_CONF = "fs.efs.key.algorithm";
+
     private String encryptedScheme;
+    private FileSystem delegate;
```

Comment:
should explain what this is and how it's different from `this.fs`.

Reply:
added in follow up